spring-integration-hazelcast/src/main/java/org/springframework/integration/hazelcast/leader/LeaderInitiator.java

```diff
@@ -252,6 +250,18 @@ public void destroy() {
 		stop();
 		this.executorService.shutdown();
 	}
+	
+	
+     FencedLock getLock() {
+		CPSubsystem cpSubSystem = client.getCPSubsystem();
```

Comment:
I guess `this.` must be added to that `client` property.

Reply:
No, it is not really needed since we already is in the "this context". But to align with the rest of the code I add it anyway ;)

Reply:
We prefer to have it to make it easier to read the code: "is that local variable or the property of the class?"